### ISSUE TITLE
bundle is broken on master

### ISSUE THREAD

**Issue by maddrag0n** [bitchcraft/injector#143]
for some reason webpack version bump fucked the bundle

when importing InjectorWebpackConfig
```js
const { InjectorWebpackConfig } = require('@bitchcraft/injector');
```

you get this error message when running webpack:
```zsh
ReferenceError: window is not defined
ReferenceError: window is not defined
    at Object.<anonymous> (…/lib/injector.es5.js:1:244)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Module.require (internal/modules/cjs/loader.js:830:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (…/webpack.config.js:5:35)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
```

**Comment by maddrag0n** [comment]
bundle manually hotfixed in release 1.1.1

**Comment by zGrav** [comment]
good bundle deployed to develop, can you also validate? if good, we push to master.

**Comment by maddrag0n** [comment]
fixed in 1.1.2
